packages/view3d/test/suite/core/PostProcessing.ts

```diff
@@ -0,0 +1,68 @@
+import { createView3D } from "../../test-utils";
+
+describe("PostProcessing", () => {
+  describe("bloom effect", () => {
+    it("should resolution be greater than or equal to canvas size", async () => {
+      const view3d = await createView3D({bloom: true});
+
+      view3d.on('load', () => {
+        const resolutionSize = view3d.postProcessing.bloomComponent.resolution;
+        const canvasSize = view3d.renderer.canvasSize;
+
+        expect(resolutionSize.width >= canvasSize.width).to.be.true;
+        expect(resolutionSize.height >= canvasSize.height).to.be.true;
+      });
```

Comment:
Is this test working?
As `src` is not used when creating a View3D instance, I think the `load` event is not called at all.
Please change the test code like the below:

```js
const view3d = await createView3D({ bloom: true });

const resolutionSize = view3d.postProcessing.bloomComponent.resolution;
const canvasSize = view3d.renderer.canvasSize;

expect(resolutionSize.width >= canvasSize.width).to.be.true;
expect(resolutionSize.height >= canvasSize.height).to.be.true;
```

If you need `ready` event to be triggered, or in other words, the View3D should be initialized, try adding a src like this:
```js
const view3d = await createView3D({ bloom: true, src: "/cube.glb" });

const resolutionSize = view3d.postProcessing.bloomComponent.resolution;
const canvasSize = view3d.renderer.canvasSize;

expect(resolutionSize.width >= canvasSize.width).to.be.true;
expect(resolutionSize.height >= canvasSize.height).to.be.true;
```

Reply:
Surprisingly, the test went well. but i'll change to that!